runtime/doc/starting.txt

```diff
@@ -545,13 +545,13 @@ accordingly.  Vim proceeds in this order:
 	":version" command.  Mostly it's "$VIM/vimrc".
 	For the Macintosh the $VIMRUNTIME/macmap.vim is read.
 
-	  *VIMINIT* *.vimrc* *_vimrc* *EXINIT* *.exrc* *_exrc* *$MYVIMRC*
+	  *NVIMINIT* *.vimrc* *_vimrc* *EXINIT* *.exrc* *_exrc* *$MYVIMRC*
```

Comment:
This breaks many tags. If you go to `runtime/doc`, run `make html`, a bunch of warnings should pop up (assuming you didn't do `-j2`, which causes no warnings to pop up for me).